docs/ai/how-to/use-redis-for-memory.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "Use Redis for Memory Storage with the Semantic Kernel SDK for .NET"
+description: "Learn how to use a Redis database with the RediSearch module for memory storage and recall in Semantic Kernel SDK for .NET."
```

Comment:
I can add an explanation about memories to the introduction. I don't think it can be expressed concisely enough for the description. But I could reword this like:

> Learn how to use a Redis database with the RediSearch module to give your AI memories in Semantic Kernel SDK for .NET.